applications/nrf_desktop/configuration/nrf52_pca20037/app_ZRelease.conf

```diff
@@ -16,6 +16,11 @@ CONFIG_DESKTOP_HID_EVENT_QUEUE_SIZE=60
 
 CONFIG_DESKTOP_BUTTONS_ENABLE=y
 
+CONFIG_DESKTOP_BUTTONS_SIM_ENABLE=y
```

Comment:
We cannot enable this simulator on any HW

Reply:
Ok, this will be part of DK keyboard.